controllers/microk8scontrolplane_controller.go

```diff
@@ -157,15 +156,8 @@ func (r *MicroK8sControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl
 			reterr = kerrors.NewAggregate([]error{reterr, err})
```

Comment:
Where is the `reterr` used?

Reply:
Was only needed for this particular section if it was required to requeue. We now requeue it after every 30 seconds so no need for it anymore. Removing them.